qa/tasks/cephfs/caps_helper.py

```diff
@@ -216,29 +206,34 @@ def run_mon_cap_tests(self, def_fs, client_id):
         else:
             raise RuntimeError('run_mon_cap_tests(): mon cap not found in '
                                'keyring. keyring -\n' + keyring)
+        return moncap
 
-        keyring_path = def_fs.admin_remote.mktemp(data=keyring)
+    def run_mon_cap_tests(self, fs, client_id):
+        """
+        Check that MON cap is enforced for a client by searching for a Ceph
+        FS name in output of cmd "fs ls" executed with that client's caps.
+
+        fs is any fs objcet so that ceph commands can be exceuted.
+        """
+        get_cluster_cmd_op = fs.mon_manager.raw_cluster_cmd
```

Comment:
I understand that this already existed - can we just use fs.mon_manager.raw_cluster_cmd(...) rather than aliasing it? IMO, nothing is gained (much) in not writing:

> fs.mon_manager.raw_cluster_cmd(...)

Reply:
Like inheriting class CephFSTestCase, this code is not added in this PR. I don't mind make these changes but it's outside the scope of this PR.